src/panels/config/automation/add-automation-element-dialog.ts

```diff
@@ -0,0 +1,555 @@
+
+  private async _fetchManifests() {
+    const manifests = {};
+    const fetched = await fetchIntegrationManifests(this.hass);
+    for (const manifest of fetched) {
+      manifests[manifest.domain] = manifest;
+    }
+    this._manifests = manifests;
+  }
+
+  protected render() {
+    if (!this._params) {
+      return nothing;
+    }
+
+    const items = this._filter
+      ? this._getFilteredItems(
+          this._params.type,
+          this._params.building_block,
+          this._group,
+          this._filter,
+          this.hass.localize,
+          this.hass.services,
+          this._manifests
+        )
+      : this._getGroupItems(
+          this._params.type,
+          this._params.building_block,
+          this._group,
+          this.hass.localize,
+          this.hass.services,
+          this._manifests
+        );
+
+    const groupName = this._group?.startsWith("service_")
+      ? domainToName(
+          this.hass.localize,
+          this._group.substring(8),
+          this._manifests?.[this._group.substring(8)]
+        )
+      : this.hass.localize(
+          // @ts-ignore
+          `ui.panel.config.automation.editor.${this._params.type}s.groups.${this._group}.label`
+        );
+
+    return html`
+      <ha-dialog open hideActions @closed=${this.closeDialog} .heading=${true}>
+        <div slot="heading">
+          <ha-header-bar>
+            <span slot="title"
+              >${this._group
+                ? groupName
+                : this.hass.localize(
+                    `ui.panel.config.automation.editor.${this._params.type}s.add`
+                  )}</span
+            >
+            ${this._group
+              ? html`<ha-icon-button-prev
+                  slot="navigationIcon"
+                  @click=${this._back}
+                ></ha-icon-button-prev>`
+              : html`<ha-icon-button
+                  .path=${mdiClose}
+                  slot="navigationIcon"
+                  dialogAction="cancel"
+                ></ha-icon-button>`}
+          </ha-header-bar>
+          <search-input
+            .hass=${this.hass}
+            .filter=${this._filter}
+            @value-changed=${this._filterChanged}
+            .label=${groupName
+              ? this.hass.localize(
+                  "ui.panel.config.automation.editor.search_in",
+                  { group: groupName }
+                )
+              : this.hass.localize(
+                  `ui.panel.config.automation.editor.${this._params.type}s.search`
+                )}
+          ></search-input>
+        </div>
+        <mwc-list
+          innerRole="listbox"
+          itemRoles="option"
+          rootTabbable
+          dialogInitialFocus
+        >
+          ${this._params.clipboardItem &&
+          !this._filter &&
+          (!this._group ||
+            items.find((item) => item.key === this._params!.clipboardItem))
+            ? html`<ha-list-item
+                twoline
+                activated
```

Comment:
Why is it activated?
Should like a bug in the dialog as we already have hover state on each item.

![CleanShot 2023-12-20 at 09 39 25](https://github.com/home-assistant/frontend/assets/5878303/bd47fb4f-e3db-4a5f-9496-622b75aaf9de)

Reply:
Let's use a divider

Reply:
And in the long run add a Toolbar to the automation editor